dso-l2/src/main/java/com/tc/objectserver/entity/EntityManagerImpl.java

```diff
@@ -245,7 +263,7 @@ public boolean removeDestroyed(EntityID id) {
       Assert.assertNotNull(service);
       Object oldService = entityServices.putIfAbsent(typeName, service);
       // This needs to be null or else there was some kind of unexpected concurrent access which would have caused failure or a duplicate entry.
-      Assert.assertNull(oldService);
+      Assert.assertNull(oldService);      
```

Comment:
There are a few extraneous whitespace changes in this commit, such as this, which should probably be kept from cluttering the version history.